script/workLoggingSystem.js

```diff
@@ -135,10 +136,54 @@ $("#submitNewTask").click(function() {
     console.log(estimatedTime);
     console.log(status);
     console.log(category);
+    if(checkValidation(priority,difficulty,status,category,estimatedTime))
+        {
+            console.log("working");
+            createNewTask(title,des,priority,difficulty,creator,assignee,estimatedTime,status,category);
+        }
+    else
+        {
+            console.log("wrong values");
+            $("#errormessage").css({"visibility": "visible"});
```

Comment:
rather than using visibility please find out the way of using show and hide class of jquery, that will be much more efficient